src/main/java/com/rtg/vcf/eval/VcfEvalCli.java

```diff
@@ -92,6 +93,7 @@ public class VcfEvalCli extends ParamsCli<VcfEvalParams> {
   protected static final String ROC_SUBSET = "roc-subset";
   protected static final String ROC_EXPR = "roc-expr";
   protected static final String ROC_REGIONS = "roc-regions";
+  protected static final String ROC_INTERSECT_MODE = "roc-intersect-mode";
```

Comment:
I'm not a fan of the flag name. Perhaps:
```suggestion
  protected static final String ROC_CROSS_JOIN = "roc-cross-join";
```

Reply:
Updated the name